app/main/views/signup.py

```diff
@@ -225,6 +225,13 @@ def render_application(id, step=None, substep=None):
         'user_email': current_user.email_address
     }
 
+    # Add service pricing
+    if 'services' in application['application']:
+        props['application']['domains'] = {}
+        for domain_name in application['application']['services'].keys():
+            response = data_api_client.get_domain_maximum_price(domain_name)
```

Comment:
couldn't you do 1 api call to get all the max prices for all domains, and then filter?

if a user has all domains thats a lot of calls...